storage/src/vespa/storage/distributor/operations/idealstate/setbucketstateoperation.cpp

```diff
@@ -34,8 +35,8 @@ SetBucketStateOperation::enqueueSetBucketStateCommand(uint16_t node, bool active
 bool
 SetBucketStateOperation::shouldBeActive(uint16_t node) const
 {
-    for (uint32_t i=0, n=_wantedActiveNodes.size(); i<n; ++i) {
-        if (_wantedActiveNodes[i] == node) {
+    for (unsigned short _wantedActiveNode : _wantedActiveNodes) {
```

Comment:
Consider `uint16_t` and avoiding leading underscore for local variable

Reply:
Fixed